recipes/frv/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "Frv" %}
+{% set version = "v4r35p6" %}
+
+# handle Virgo version number format
+{% set semver_version = version|
+    replace('v', '', 1)|
+    replace('r', '.', 1)|
+    replace('p', '.', 1)
+ %}
```

Comment:
Ouch!

Reply:
Indeed, this seems to be a legacy of the CMT build system used in high-energy physics back in the 90s.